firmware/application/apps/ui_playlist.cpp

```diff
@@ -158,7 +173,7 @@ void PlaylistView::start() {
     auto p = std::make_unique<FileReader>();
     auto open_error = p->open(file_path);
     if (open_error.is_valid()) {
-        file_error();
+        file_error("illegal grammar(176)");
```

Comment:
Nit: putting the line number in the message is really brittle because one line add/Delete will make it wrong.

Reply:
I see 176 and 255 have the same error message. They should be different, or if they are the same, they need to be broken out into their own common function

Reply:
Thank you! changed.